recipes-ni/docker-functional-tests/docker-functional-tests.bb

```diff
@@ -0,0 +1,37 @@
+SUMMARY = "Functional test of docker container capabilities on NILRT"
+HOMEPAGE = "https://docker.com"
+SECTION = "tests"
+LICENSE = "GPL-2.0-only & GPL-2.0-or-later"
+LIC_FILES_CHKSUM = "file://${COMMON_LICENSE_DIR}/GPL-2.0-or-later;md5=fed54355545ffd980b814dab4a3b312c"
+
+inherit ptest
+
+FILESEXTRAPATHS:prepend := "${THISDIR}/files:"
```

Comment:
The `FILESEXTRAPATHS` trick is only needed in `.bbappend` files. The recipe-local `files/` directory should already be in the PATHS for a `.bb` recipe.